File: javacode/src/test/java/com/example/LiteralTest.java
package com.example;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class LiteralTest {
  @Test
  public void evaluate5() {
    Literal literal = new Literal(5);
    assertEquals(5, literal.evaluate());
  }

  @Test
  public void evaluate42() {
    Literal literal = new Literal(42);
    assertEquals(42, literal.evaluate());
  }
}

File: csharpcode/ExpressionEvaluator.Tests/LiteralTest.cs
namespace ExpressionEvaluator.Tests;

public class LiteralTests
{
    [Test]
    public void Evaluate5()
    {
        Literal literal = new Literal(5);
        Assert.That(literal.Evaluate(), Is.EqualTo(5));
    }

    [Test]
    public void Evaluate42()
    {
        Literal literal = new Literal(42);
        Assert.That(literal.Evaluate(), Is.EqualTo(42));
    }
}
